pkg/crc/oc/oc.go

```diff
@@ -25,6 +26,7 @@ type OcConfig struct {
 type OcLocalRunner struct {
 	OcBinaryPath   string
 	KubeconfigPath string
+	Context        string
```

Comment:
Have you considered putting this in `OcConfig` rather than `OcLocalRunner`? (but then `KubeconfigPath` would belong there too). Juts random thoughts at this point to avoid duplicating too much between `oc.go` and `cert-renewal.go`